packages/web-worker/src/config.js

```diff
@@ -0,0 +1,28 @@
+const { schema } = require('@bugsnag/core/config')
+const map = require('@bugsnag/core/lib/es-utils/map')
+const assign = require('@bugsnag/core/lib/es-utils/assign')
+
+module.exports = {
+  appType: {
+    ...schema.appType,
+    defaultValue: () => 'worker-js'
```

Comment:
this is all one word in other notifiers, e.g. `browserjs` or `electronrendererjs`

```suggestion
    defaultValue: () => 'workerjs'
```